site/styles/_cta-buttons.scss

```diff
@@ -1,4 +1,16 @@
 /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+@mixin text-cta-btn-inline {
+  margin: 0;
+  padding: 6px 12px;
+  width: 100%;
```

Comment:
Nitpick: This 100% width directly contradicts the name of the mixin.

Reply:
Changed to text-cta-btn